presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -43,6 +43,7 @@
 import static io.airlift.slice.SizeOf.SIZE_OF_LONG;
 import static io.airlift.slice.SizeOf.SIZE_OF_SHORT;
 import static io.airlift.slice.SizeOf.sizeOf;
+import static io.airlift.slice.Slices.EMPTY_SLICE;
```

Comment:
this belongs to the other commit I guess, it's confusing not to see it being used in this commit.